.gitmodules

```diff
@@ -809,3 +809,6 @@
 [submodule "vendor/grammars/reason"]
 	path = vendor/grammars/reason
 	url = https://github.com/facebook/reason
+[submodule "vendor/grammars/sublime-mzn"]
+	path = vendor/grammars/sublime-mzn
+	url = https://github.com/astenmark/sublime-mzn
```

Comment:
Oh right! Let's keep the Sublime Text one then and in the future we'll switch if the Atom one receives more contributions.